images/linux/scripts/installers/configure-environment.sh

```diff
@@ -25,3 +25,7 @@ chmod -R 777 $AGENT_TOOLSDIRECTORY
 # https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html
 # https://www.suse.com/support/kb/doc/?id=000016692
 echo 'vm.max_map_count=262144' | tee -a /etc/sysctl.conf
+
+# Create symlink for tests running
+chmod -R 777 $HELPER_SCRIPTS/invoke-tests.sh
+ln -s $HELPER_SCRIPTS/invoke-tests.sh /usr/local/bin
```

Comment:
may be name it without `sh`? just `invoke-tests`. In this case, it will look like `invoke_tests` that is more beatiful

Reply:
Done